dependencies.props

```diff
@@ -50,6 +50,10 @@
     <XunitNetcoreExtensionsVersion>2.2.0-preview1-02928-01</XunitNetcoreExtensionsVersion>
 
     <!-- Roslyn optimization data package version -->
+    <MicrosoftDotNetIBCMergePackageVersion>4.6.0-alpha-00001</MicrosoftDotNetIBCMergePackageVersion>
+    <TestILCamd64retPackageVersion>$(ProjectNTfsTestILCPackageVersion)</TestILCamd64retPackageVersion>
```

Comment:
I don't see the value since we generally take all three version updates at once.   Either way, since you're adding these Properties, it'd be preferable to PascalCase them.

Reply:
I'm using the typical pattern here of using the package name, removing the `.`s from it, and then appending `PackageVersion` to make the property name.

https://github.com/dotnet/corefx/blob/84883052ef0f720a8befdc8525db50c9544227d5/external/test-runtime/optional.json#L6

`TestILC.amd64ret` becomes `TestILCamd64retPackageVersion`.

Reply:
Prodcon added a little extra rule because `runtime` packages (especially ones like `transport.runtime.osx-x64.Microsoft.NETCore.Runtime.CoreCLR`) are ugly/hard to read that way:

> The property names are determined based on each package identity and converted to typical MSBuild convention:
>  * Any non-alphanumeric character (e.g. ".", "_") is considered a delimiter, and is removed.
>  * The first letter of each segment is uppercased to match MSBuild PascalCase convention. (Any existing capital letters remain capitalized.)

([dependency-uptake.md](https://github.com/dotnet/core-eng/blob/master/Documentation/Orchestrated-Build/Api/dependency-uptake.md))